Arduino-Bling/RobotBling/animations.ino

```diff
@@ -40,3 +40,33 @@ void blinkOffWithDelay(uint8_t wait)
   setAllPixelsOff();
   delay(wait);
 }
+
+ void strobe(rgbColor c, uint8_t numFlashes, int flashDelay, int strobePause)
```

Comment:
This comment is a general reference to use of `int`. In embedded development (such as Arduino), it is a good practice to be explicit about data types. This helps with portability and memory management (basically how much of the memory we are using). 'int' will depend on the hardware system and can be 8, 16, or 32 bits long. Types such as `uint8_t` is very explicit: it is an unsigned (meaning only positive values starting with 0), 8 bits (meaning 11111111) which translates to a number between 0 to 255. 

Given the delay values will not be more than 255, pls change int to uint8_t where applies.